### ISSUE TITLE
AssertionError: TensorRT plugin should be compiled

### ISSUE THREAD

**Issue by Enverrrr** [open-mmlab/mmdeploy#1031]
Hey I am developing mmocr on nvidia jetson nano. When I try to onnx2tensorrt deployment with the psnet_ctw detection model, I take this error. I read the documents about this issue but I couldn't find anything. 

![hata](https://user-images.githubusercontent.com/60448321/189591502-7e8b22d9-d9bd-4aa6-8d11-6670f96166e7.png)

**Comment by tpoisonooo** [comment]
Please `python3 tools/check_env.py` and find tensorrt item. 

If it shows `none`, follow [this tutorial](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/01-how-to-build/jetsons.md) and install pytensorrt and pycuda.

**Comment by lvhan028** [comment]
Hi, @Paschapy 
Deployment tools in mmocr are already deprecated now. We encourage users to use mmdeploy instead.

**Comment by Paschapy** [comment]
> Hi, @Paschapy Deployment tools in mmocr are already deprecated now. We encourage users to use mmdeploy instead.

Okey I got it. But I couldn't installation on jetson nano, when I tried to install mmdeploy I took an error. I looking for solution only I think it is unsolutions. 
My currently problem is python version (3.6.9) in Jetpack SDK. I tried to uninstall python3.6.9 and my jetson broken, I tried archiconda and I take pip3 error.

**Comment by Paschapy** [comment]
> Please `python3 tools/check_env.py` and find tensorrt item.
> 
> If it shows `none`, follow [this tutorial](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/01-how-to-build/jetsons.md) and install pytensorrt and pycuda.

Hi I made progress, I finished installation. And I ran tools/deploy.py I took a different error. 
Like that;
![error](https://user-images.githubusercontent.com/60448321/190377996-2ba8d8da-5a52-41bd-84c3-fae77a89b645.png)

I have no idea what to do for solution.

**Comment by lvhan028** [comment]
@AllentDan could you help investigate it?

**Comment by AllentDan** [comment]
You have to put icdar2015 dataset into a relative folder data .

**Comment by Paschapy** [comment]
> 

Thank you for recommendation, I did it but took a different error like this,
Traceback (most recent call last):
  File "tools/deploy.py", line 12, in <module>
    from mmdeploy.apis import (create_calib_input_data, extract_model,
ImportError: cannot import name 'create_calib_input_data'

When I made a progress, I'm getting back.
I formatted the jetson nano, and I followed the installation docs. But it's not worked.
Maybe problem is my OS so Jetpack SDK 4.6.1 Jetson Nano version. I don't know.
